### PR TITLE
Auth6

### PR DESCRIPTION
**Pull request by carvilsi**
Handling sigUp error
Modifying logger to take logging default var

### CODE REVIEW

File: src/auth/AuthCtrl.coffee [hunk 1]
```diff
@@ -48,6 +49,12 @@ validateJSONSchema = (jsonReq, jsonSch) ->
   v = new Validator()
   v.validate(jsonReq,jsonSch).valid
   
+errorCodeParserFlock = (res) ->
+  # For signUp error cases
+  if res.statusCode is 409
+    Promise.reject(Error WeaverError.DUPLICATE_VALUE, 'DUPLICATION ERROR, USERNAME OR EMAIL IS ALREADY TAKEN')
```
Comment (malamili) [inline src/auth/AuthCtrl.coffee]: The messages can just be normal human readable, so just write Duplication error, .... Also, can you add context information like supplied username or email? If too difficult, dont bother,

File: src/util/logger.coffee [hunk 2]
```diff
@@ -28,6 +29,6 @@ Logger = winston.Logger
 logger = new Logger({transports})
 
 # Set minimum level
-logger.level = 'error'
+logger.level = "#{config.get('logging.console')}".toLowerCase()
```
Comment (malamili) [inline src/util/logger.coffee]: Nice!!

File: src/auth/AuthCtrl.coffee (hunk 1 above)
Comment (carvilsi) [inline src/auth/AuthCtrl.coffee]: This feature was already at the previous commit, but the answer of weaver-flock changes to less explicit:
+errorCodeParserFlock = (res) ->
 +  # For signUp error cases
 +  if res.statusCode is 500
 +    if !!~ res.error.Error.message.indexOf "userName"
 +      Promise.reject(Error WeaverError.USERNAME_TAKEN, 'USERNAME_TAKEN')
 +    else if !!~ res.error.Error.message.indexOf "userEmail"
 +      Promise.reject(Error WeaverError.EMAIL_TAKEN, 'EMAIL_TAKEN')
 +  else
 +    Promise.reject(Error WeaverError.OTHER_CAUSE, 'OTHER_CAUSE')

File: src/util/logger.coffee (hunk 2 above)
Comment (carvilsi) [inline src/util/logger.coffee]: 👍

**Pull request merged**